runtime/src/main/java/org/corfudb/runtime/view/SequencerView.java

```diff
@@ -1,6 +1,8 @@
 package org.corfudb.runtime.view;
 
-import java.util.Set;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)